api/app/Models/User.php

```diff
@@ -158,4 +158,33 @@ protected function generateSubject()
         else
             return true;
     }
+   public function getProfileStatusAttribute(): bool
+    {if(is_null($this->attributes['first_name']) Or
+        is_null($this->attributes['last_name']) Or
+        is_null($this->attributes['email']) Or
+        is_null($this->attributes['telephone']) Or
+        is_null($this->attributes['preferred_lang']) Or
+        is_null($this->attributes['current_province']) Or
+        is_null($this->attributes['current_city']) Or
+        is_null($this->attributes['looking_for_english']) Or
+        (is_null($this->attributes['looking_for_english']) &&
+        is_null($this->attributes['looking_for_french']) &&
+        is_null($this->attributes['looking_for_bilingual'])) Or
+        is_null($this->attributes['is_gov_employee']) Or
+        is_null($this->attributes['location_preferences']) Or
+        is_null($this->attributes['would_accept_temporary']) Or
+        is_null($this->attributes['is_woman']) Or
+        is_null($this->attributes['has_disability']) Or
+        is_null($this->attributes['is_indigenous']) Or
+        is_null($this->attributes['is_visible_minority']) Or
```

Comment:
Nice, got the backend up!
Quick observation though, should equity fields be involved in this check? I don't think they should, that'd make an odd form would it not?

Reply:
I am not sure. waiting for reviews and comments. I am thinking after these fields are filled only user can apply for the the job. There are some other fields I am not sure about. Thank you for the comment. Hope it will help to finalize the required fields.

Reply:
I agree with Vachan.  Please double-check the fields you are using to determine if the profile is complete.

Reply:
I just checked all the required fields in all the forms but equity form is not finalized yet. Any other smart way to determine the required fields? @petertgiles So user can apply for the job wihout filling the equity form?

Reply:
You could check the forms on each profile section page to see which fields are required.  If you're not sure after that you could check with Jerbo.